src/chains/details/gnosis.ts

```diff
@@ -0,0 +1,57 @@
+import { ChainInfo, SupportedChainId } from '../types'
+import { nativeCurrencyTemplate, WRAPPED_NATIVE_CURRENCIES } from '../../common/consts/tokens'
+import { RAW_CHAINS_FILES_PATH } from '../const'
+
+const gnosisChainLogo = `${RAW_CHAINS_FILES_PATH}/images/gnosis-logo.svg`
+
+/**
+ * Gnosis chain chain info.
+ *
+ * See also https://github.com/wevm/viem/blob/main/src/chains/definitions/gnosis.ts
+ */
+export const gnosisChain: ChainInfo = {
+  id: SupportedChainId.GNOSIS_CHAIN,
+  label: 'Gnosis Chain',
+  nativeCurrency: {
+    ...nativeCurrencyTemplate,
+    chainId: SupportedChainId.GNOSIS_CHAIN,
+    name: 'xDAI',
+    symbol: 'xDAI',
+    logoUrl: WRAPPED_NATIVE_CURRENCIES[SupportedChainId.GNOSIS_CHAIN].logoUrl,
+  },
+  addressPrefix: 'gno',
+  isTestnet: false,
+  contracts: {
+    multicall3: {
+      address: '0xca11bde05977b3631167028862be2a173976ca11',
+      blockCreated: 21022491,
+    },
+  },
+  rpcUrls: {
+    default: {
+      http: ['https://rpc.gnosischain.com'],
+      webSocket: ['wss://rpc.gnosischain.com/wss'],
+    },
+  },
+  color: '#07795B',
+  logo: { light: gnosisChainLogo, dark: gnosisChainLogo },
```

Comment:
I know this is a mistake I did, but doesn't hurt to suggest.
Refactor idea: update logo type to: `type logo: string | { light: string, dark: string }`
This way we don't need to use the same logo twice when there's no difference.

Reply:
I would leave it like this for simplicity, requires type assertions to show it to save some bytes, I was also considering to model as `default` and then allow to add arbitrary properties with other themes, but I also thought I don't want to overcomplicate. All these things kind of fall out of the scope and will come naturally as we organically need them

If you really want, I can change it, not big deal to me.

Reply:
No no, just pondering over my initial decision, but it's ok as is.